resources/scripts/components/server/files/FileManagerContainer.tsx

```diff
@@ -71,12 +82,20 @@ export default () => {
                             />
                         }
                     />
+                    <input
+                        onChange={searchFiles}
+                        css={tw`rounded-lg bg-neutral-700 border-2 border-cyan-700 md:mx-6 w-full px-4 mb-4 md:mb-0`}
+                        placeholder='search'
+                        style={{ borderColor: 'rgb(59 130 246);' }}
```

Comment:
This shouldn't be inline styled, in addition to this input bar not matching the rest of the application's inputs.